packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metriport/core",
-  "version": "1.3.0-alpha.31",
+  "version": "1.3.0-alpha.1",
```

Comment:
was this intentional? looks like it's going back a version

Reply:
Ditto diff branches work8ng on the same next version